layouts/default.vue

```diff
@@ -112,45 +112,96 @@
 </template>
 
 <script lang="ts">
+import { stringify } from 'qs';
 import Vue from 'vue';
 import { mapActions, mapState } from 'vuex';
 
+interface WebSocketMessage {
+  MessageType: string;
+  Data?: never;
+}
+
 export default Vue.extend({
   data() {
     return {
       drawer: true,
-      opacity: 0
+      opacity: 0,
+      keepAliveInterval: undefined as number | undefined
```

Comment:
it is not ideal, but we could infer type here
```
      keepAliveInterval: setTimeout(() => {}, 0)
```
Alternatively, we could do:
```
      keepAliveInterval: undefined as typeof setInterval | undefined
```
and on L176 do:
```
    if (typeof this.keepAliveInterval === 'number') {
```
However this causes TS errors on L190